hoot-core-test/src/test/cpp/hoot/core/language/HootServicesLanguageDetectorClientTest.cpp

```diff
@@ -41,6 +41,9 @@
 #include <hoot/core/util/UuidHelper.h>
 #include <hoot/core/language/LanguageDetectionConfidenceLevel.h>
 
+// Qt
+#include <QNetworkCookie>
```

Comment:
This is a lot of adding `#include <QNetworkCookie>`.  Is there a better way to do that?  It isn't directly used in those source files.  Should it be included in the header file that it is used in?  That might solve all of these changes.  Just an idea.

Reply:
Yes, that was ugly. Fixed.